src/applications/static-pages/BTSSS-login/App/index.jsx

```diff
@@ -0,0 +1,29 @@
+// Node modules.
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { isLoggedIn } from 'platform/user/selectors';
+import AuthContext from '../AuthContext';
+import UnauthContext from '../UnauthContext';
+
+export const App = ({ currentlyLoggedIn, BTSSSLoginWidget }) => {
+  if (BTSSSLoginWidget) {
+    return currentlyLoggedIn ? <AuthContext /> : <UnauthContext />;
+  }
+  return <></>;
+};
+
+const mapStateToProps = state => ({
+  currentlyLoggedIn: isLoggedIn(state),
+  BTSSSLoginWidget: state?.featureToggles?.BTSSSLoginWidget,
```

Comment:
**BLOCKING**
This will not work. We need to do one of the following:
1. Reference the feature toggle by the name it has in vets-api, either camelCased or snake_cased:
  - `state?.featureToggles?.btsssLoginWidget`
  - `state?.featureToggles?.btsss_login_widget`
2. Use the selector that was also created as part of this PR. Since it was created, this might be the best path forward. If that feels like overkill (which is arguable), then we should remove it. Currently, though, it seems like this line is conflating the two approaches.

Reply:
I opted for a mix of option 1 & 2 after changing how the selector works since I couldn't get it to receive a state object as a parameter (it was always undefined). The approach I used follows the [Platform docs on the subject](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-8.Usethefeaturetogglevaluetogateyournewbehavior), and I could see it in use in other react components as well (though I did also see createSelector() in many places as well).